packages/amplify-cli-core/API.md

```diff
@@ -113,6 +113,32 @@ export abstract class AmplifyCategoryTransform {
     abstract transform(context: $TSContext): Promise<Template>;
 }
 
+// @public (undocumented)
+export const amplifyCLIConstants: Readonly<{
+    readonly AmplifyCLIDirName: "amplify";
+    readonly DotAmplifyDirName: ".amplify";
+    readonly DotConfigAmplifyCLISubDirName: ".config";
+    readonly BackendAmplifyCLISubDirName: "backend";
+    readonly CurrentCloudBackendAmplifyCLISubDirName: "#current-cloud-backend";
+    readonly ProjectConfigFileName: "project-config.json";
+    readonly amplifyMetaFileName: "amplify-meta.json";
+    readonly CliName: "amplify";
+    readonly DateTimeFormatString: "YYYY-MM-DD-HH-mm-ss";
+    readonly DateTimeFormatStringCompact: "YYYYMMDDHHmmss";
+    readonly DefaultAWSAccessKeyId: "<YOUR_ACCESS_KEY_ID>";
+    readonly DefaultAWSSecretAccessKey: "<YOUR_SECRET_ACCESS_KEY>";
+    readonly DefaultAWSRegion: "us-east-1";
+    readonly LocalEnvFileName: "local-env-info.json";
+    readonly ProviderInfoFileName: "team-provider-info.json";
+    readonly BackendConfigFileName: "backend-config.json";
+    readonly TagsFileName: "tags.json";
+    readonly MIN_MIGRATION_PROJECT_CONFIG_VERSION: "3.0";
+    readonly MIN_NODE12_PROJECT_CONFIG_VERSION: "3.1";
+    readonly CURRENT_PROJECT_CONFIG_VERSION: "3.1";
+    readonly BreadcrumbsFileName: "amplify.state";
+    readonly LogDirectory: ".amplify-log";
+}>;
```

Comment:
could we fix casing of properties? it' mixed.

Reply:
I want to avoid renamings in these migration PRs. If we proceed with moving toolkit in this way, it will end up being a large change. I want to minimize the ripples as much as possible.

That said, we do have inconsistent cases in a few places across the repository. I think a casing lint might be a good idea to help get us in a more consistent state.